github/hooks_config.go

```diff
@@ -0,0 +1,16 @@
+package github
+
+// HookConfig describes metadata about a webhook configuration.
```

Comment:
Instead of creating a file with only one struct in it, let's please move this to `repos_hooks_configuration.go` where it is most heavily used.